packages/syft/src/syft/service/worker/worker_pool_service.py

```diff
@@ -178,6 +179,109 @@ def delete_worker(
             message=f"Worker with id: {worker_id} deleted successfully from pool: {worker_pool.name}"
         )
 
+    @service_method(
+        path="worker_pool.get_worker",
+        name="get_worker",
+        roles=DATA_OWNER_ROLE_LEVEL,
+    )
+    def get_worker(
+        self, context: AuthedServiceContext, pool_name: str, worker_id: UID
```

Comment:
All the interfaces , currently use worker_pool_id, 

like delete_worker and get_worker_logs

later we could  add  worker_pool_name, if we require later.